### PR TITLE
OTC-67: RFC-116 New state of policies

### PR DESCRIPTION
**Pull request by bartekkaminski**
Added ActivationOption for Policies

TASK:
https://openimis.atlassian.net/browse/OTC-67

### CODE REVIEW

File: IMIS_DAL/UtilitiesDAL.vb [hunk 1]
```diff
@@ -132,4 +132,18 @@ Public Class UtilitiesDAL
 
     End Sub
 
+    Public Function getActivationOption() As Integer
```
Comment (dragos-dobre) [inline IMIS_DAL/UtilitiesDAL.vb:135]: There is the IMISDefaultsDAL dealing with the IMISDefaults table. Move this function there and make the necessary changes elsewhere.

File: IMIS_BL/EscapeBL.vb [hunk 2]
```diff
@@ -81,4 +82,10 @@ Public Class EscapeBL
         Return BulkSMSId
 
     End Function
+
+    Public Function getActivationOption() As Integer
```
Comment (dragos-dobre) [inline IMIS_BL/EscapeBL.vb:86]: If the value is in DB, we might not need this function here. Make a note to remove it after we develop the other RFCs and it is not used.

File: IMIS/App_GlobalResources/Resource.resx [hunk 3]
```diff
@@ -4889,4 +4889,7 @@ adults/children)</value>
 	<data name="M_NOMESSAGETOBEDESPLAYED" xml:space="preserve">
 		<value>No Message to be displayed</value>
 	</data>
+  <data name="T_READY" xml:space="preserve">
```
Comment (dragos-dobre) [inline IMIS/App_GlobalResources/Resource.resx:4892]: Did you add this in Lokalize?

File: IMIS_BL/EscapeBL.vb (hunk 2 above)
Comment (bartekkaminski) [reply IMIS_BL/EscapeBL.vb:86]: The method is not used at this time. If the method is not needed at the end of the ePayment implementation, I will remove it.

File: IMIS_DAL/UtilitiesDAL.vb (hunk 1 above)
Comment (bartekkaminski) [reply IMIS_DAL/UtilitiesDAL.vb]: I moved the `getActivationOption()` method to the **IMISDefaultsDAL**

File: IMIS/App_GlobalResources/Resource.resx (hunk 3 above)
Comment (bartekkaminski) [reply IMIS/App_GlobalResources/Resource.resx:4892]: Yes, I added `T_READY` to Lokalise

**Pull request merged**